.npmignore

```diff
@@ -14,7 +14,7 @@ coverage
 
 #editor settings
 .vscode
+.idea
 
 #src
-src
```

Comment:
Please readd src here, (somehow I cannot change this myself in this PR)

Reply:
Please readd src here, (somehow I cannot change this myself in this PR)